packages/artillery/lib/platform/aws-ecs/worker/Dockerfile

```
@@ -2,34 +2,57 @@
 # NOTE: Version we use here needs to be kept consistent with that in
 # artillery-engine-playwright.
 # ********************************
-FROM mcr.microsoft.com/playwright:v1.43.1
-LABEL maintainer="team@artillery.io"
+FROM mcr.microsoft.com/playwright:v1.43.1 as build-image
```

```suggestion
FROM mcr.microsoft.com/playwright:v1.43.1
```

Doesn't seem to be a multi-stage (yet?)